packages/integration-tests/suites/replay/errors/template.html

```diff
@@ -5,7 +5,7 @@
   </head>
   <body>
     <button id="go-background">Go to background</button>
-    <button id="error">Throw Error</button>
+    <div role="button" id="error" class="btn btn-error" aria-label="Throw Error">Throw Error</div>
```

Comment:
Since the errorMode and errorsInSession tests were already capturing this breadcrumb, decided to add attributes here and make sure they were captured.